### ISSUE TITLE
Hyperparameter Tuning MFNN

### ISSUE THREAD

**Issue by materialsguy** [lululxvi/deepxde#207]
Hi Lu,

I'm currently on the way to tune my hyperparameters. I had a look at the documentation but I wasn't able to find any learning rate finder or something similar. Is there already something implemented that I just didn't see? Do you have any tips on how to tune hyperparameters of the MFNN in general or is it basically educated trial and error?

Thanks.

Best,
Claus

**Comment by lululxvi** [comment]
What do you mean by "learning rate finder"? Tuning hyperparameters is mostly trial and error, or you can do this automatically, e.g., random search etc.
